ddp-workspace/projects/ddp-testboston/src/assets/i18n/en.json

```diff
@@ -67,7 +67,7 @@
       "Title": "We want to help our communities and learn more about COVID-19 infection patterns in the Boston area.",
       "Text": "TestBoston is a COVID-19 study supported by Brigham and Women's Hospital and the Broad Institute of MIT and Harvard.",
       "Code": "I have an invitation code",
-      "NoCode": "I don’t have an invitation code and want to join the study"
+      "NoCode": "We are currently fully enrolled.  Please check back in for future updates!"
```

Comment:
Might be better to introduce a new translation variable? What about other languages?

Reply:
Ah, I see your point--if we leave `NoCode` as is, the translations will be **wrong** (as in the meaning is totally different).  I've updated this so it uses a new variable, so that at least other languages will see the untranslated value instead of the text linking them to MGB.